paasta_tools/paasta_cli/utils.py

```diff
@@ -436,12 +436,17 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         app_id_flag = "--appid %s " % kwargs['app_id']
     else:
         app_id_flag = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s' % (
+    if 'delta' in kwargs and kwargs['delta']:
+        delta = "--delta %s" % kwargs['delta']
+    else:
+        delta = ''
+    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
```

Comment:
I'm not sure about this - ``emergency_scale`` is a marathon only tool, but ``paasta_serviceinit`` doesn't have any validation to prevent it being called on a Chronos job. This doesn't map to anything useful for chronos, so we need to add some validation somewhere for this. I'm not sure where is the best place right now though.